frontend/src/Crossword.tsx

```diff
@@ -415,9 +417,9 @@ const Crossword: React.FC<CrosswordProps> = (props) => {
     canvas.onmousedown = handleMouseDown;
     canvas.onmousemove = handleMouseMove;
     canvas.onmouseup = handleMouseUp;
-    canvas.ontouchstart = handleTouchStart;
-    canvas.ontouchmove = handleTouchMove;
-    canvas.ontouchend = handleTouchEnd;
+    canvas.addEventListener("touchstart", handleTouchStart)
+    canvas.addEventListener("touchmove", handleTouchMove)
+    canvas.addEventListener("touchend", handleTouchEnd)
```

Comment:
Add return statement where the event listeners are removed. Otherwise they will stay attached even after the component is unmounted.